portal/src/layout/header/useNavigationLinks.tsx

```diff
@@ -200,7 +209,7 @@ export function useNavigationLinks(): NavigationLinks {
                         basePath: PageType.KOMPONENTER,
                     })),
                 {
-                    linkText: "React Hooks",
+                    linkText: pageNames[PageType.KOMPONENTER],
```

Comment:
Jeg fanget ikke denne, men navnet er ikke Komponenter her. Tenker vi kan la denne være siden den ikke er sin egen `PageType`.

```suggestion
                    linkText: "React Hooks",
```